app/main/views/users.py

```diff
@@ -76,15 +82,18 @@ def list_users():
 
         user_query = user_query.filter(User.supplier_id == supplier_id)
 
-    users = user_query.paginate(
+    paginated_users = user_query.paginate(
         page=page,
         per_page=current_app.config['DM_API_SERVICES_PAGE_SIZE'],
     )
 
     return jsonify(
-        users=[user.serialize() for user in users.items],
+        users=[user.serialize() for user in paginated_users.items],
+        meta={
+            "total_pages": paginated_users.pages
```

Comment:
Is `total_pages` the same as `last_page`? Elsewhere where we do pagination we have links for previous, next and last pages. I guess a count is different from a URL, but not sure it is necessary?